cmd/validator-exporter/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+
+	"github.com/caarlos0/env/v10"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/archway-network/validator-exporter/pkg/collector"
+	"github.com/archway-network/validator-exporter/pkg/config"
+	"github.com/archway-network/validator-exporter/pkg/grpc"
+	log "github.com/archway-network/validator-exporter/pkg/logger"
+)
+
+func main() {
+	port := flag.Int("p", 8008, "Server port")
+	logLevel := log.LevelFlag()
+
+	flag.Parse()
+
+	log.SetLevel(*logLevel)
+
+	cfg := config.Config{}
+	if err := env.Parse(&cfg); err != nil {
+		log.Fatal(err.Error())
+	}
+
+	_, err := grpc.LatestBlockHeight(cfg)
```

Comment:
What is this for and why do we need the LatestBlockHeight?

Reply:
This is to check if provided GRPC_ADDR is a working by calling one of its method. If it doesn't work exporter fails on startup.